src/api/spec/models/staging/staging_project_spec.rb

```diff
@@ -159,6 +159,7 @@
   describe '#assign_managers_group' do
     context 'when the group wasn\'t assigned before' do
       before do
+        User.current = user
```

Comment:
why not `login(user)`? :thinking:

Reply:
I guess I can do that. - but I found User.current in tons of model tests.